.gitignore

```diff
@@ -3,7 +3,7 @@ node_modules/
 yarn.lock
 package-lock.json
 *.log
-
+static/
```

Comment:
Shouldn't ignore static/ if *.scss source files are located there
@alvinsiu shed your opinion here on the intention behind static/, staticfiles/...?

Reply:
:lady_beetle: I agree, `static/` should not be added to `.gitignore` or it will ignore all non-app-specific static files. We should instead ignore `staticfiles/` or whatever is configured for `STATIC_ROOT`.

If you want to specifically have `static/` be the folder for compiling static assets into, that warrants a discussion since that would differ significantly from most of the apps we've built.